build.rs

```diff
@@ -0,0 +1,187 @@
+use bzip2::read::BzDecoder;
+use std::{
+    env,
+    fs::{create_dir_all, write as fs_write, File},
+    path::{Path, PathBuf},
+    process::Command,
+};
+use tar::Archive;
+use zip::ZipArchive;
+
+const TESTGEN_HS_PATH: &str = "TESTGEN_HS_PATH";
+
+fn main() {
+    if env::var(TESTGEN_HS_PATH).is_ok() {
```

Comment:
Now I thought that if the user sets this variable, we don’t verify whether they gave us the correct version. :eyes: But then maybe this verification should be done in runtime, by the `blockfrost-platform` executable?

Reply:
Moved to:
* #115.